drivers/dma/Kconfig.arc_hs

```
@@ -0,0 +1,9 @@
+# Copyright (c) 2025 Tenstorrent AI ULC
+# SPDX-License-Identifier: Apache-2.0
+
+config DMA_ARC_HS
+	bool "ARC DMA driver"
+	default y
+	depends on DT_HAS_SNPS_DESIGNWARE_DMA_ARC_HS_ENABLED && DMA
```

The `&& DMA` can be dropped because of the `if DMA` block in `drivers/dma/Kconfig`

```suggestion
	depends on DT_HAS_SNPS_DESIGNWARE_DMA_ARC_HS_ENABLED
```